source/elements/oneDNN/source/api/primitives.rst

```diff
@@ -50,4 +50,21 @@ On the logical level, the library provides the following abstractions:
    :maxdepth: 1
```

Comment:
(Random spot). There are API subsections in the TOC that I think should not be there:
![image](https://user-images.githubusercontent.com/15225631/75933645-5aea4d80-5e2f-11ea-909f-74388da1a413.png)